code/game/objects/items/devices/uplink_random_lists.dm

```diff
@@ -65,7 +65,6 @@ var/datum/uplink_random_selection/default_uplink_selection = new/datum/uplink_ra
 	items += new/datum/uplink_random_item(/datum/uplink_item/item/tools/space_suit, 50, 10)
 	items += new/datum/uplink_random_item(/datum/uplink_item/item/tools/thermal)
 	items += new/datum/uplink_random_item(/datum/uplink_item/item/tools/heavy_vest)
```

Comment:
```suggestion
	items += new/datum/uplink_random_item(/datum/uplink_item/item/tools/heavy_vest)
	items += new/datum/uplink_random_item(/datum/uplink_item/item/tools/mercarmor)
```
Might as well add the new box to the list.

Reply:
I tried that and it didn't work for some reason. I did have an issue that I fixed afterward, so it might be fixed now, but I need to check